### PR TITLE
mario enhancements

### PR DESCRIPTION
**Pull request by meg-1**
_**adding a few enhancements to the Mario example for better UI and usage.**_
- fixing a bug where a snippet of code accidentally displayed on the main page
- fixing the PyScript code and tag indentation
- adding styles for the simple instructions to be visible

### CODE REVIEW

**Review by marimeireles — CHANGES_REQUESTED** [review]
Besides this little nitpick... For some reason the box where the video should be showing up is just a blue box on my screen.
So I think that's broken?
Maybe because of some indentation issue.

**Comment by marimeireles** [comment]
This is such a complicated example though.
I'd really like to move it from here to the collective.
Also, I'm available for a quick check up on the status of the collective, like 30min. Maybe we should do that some point next week? There are a lot of lose ends I think it'd be better discussed live rather than through text.
Thoughts @fpliger, @pauleveritt?

**Comment by marimeireles** [comment]
Oh, and thank you for tackling this @meg-1! :)
Let's see what's the future of this example, it's a complex one and we're thinking about just having the simpler ones in this repo, but anyways it'd be nice to have them with a better presentation.
Thanks and cheers.

**Comment by pauleveritt** [comment]
Hi @meg-1 thanks for putting work in on this. As @marimeireles we're working to get something off the ground in the "Collective". It will mean some refactoring and some test writing. If you're up for it, we can work together and help define some of the standards.

File: examples/mario/play_mario.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
     <title>Svelte app</title>
 
     <link rel="icon" type="image/png" href="../favicon.png" />
-    <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
+    <link rel="stylesheet" href="../https://pyscript.net/alpha/pyscript.css" />
+    <link rel="stylesheet" href="css/game.css" />
```
[the remark this answers is not in the document]
Comment (meg-1) [reply examples/mario/play_mario.html]: thank you for your feedback! just rolled back the changes on the two lines as requested :)

**Comment by meg-1** [comment]
> Oh, and thank you for tackling this @meg-1! :) Let's see what's the future of this example, it's a complex one and we're thinking about just having the simpler ones in this repo, but anyways it'd be nice to have them with a better presentation. Thanks and cheers.

thank you so much for your answers!! apologies for responding so late :') and of course, I understand! sounds like a sound decision. only glad to help out as much as i can :)

**Comment by meg-1** [comment]
> Hi @meg-1 thanks for putting work in on this. As @marimeireles we're working to get something off the ground in the "Collective". It will mean some refactoring and some test writing. If you're up for it, we can work together and help define some of the standards.

hi, @pauleveritt ! sorry for such a late reply :') but thank you for answer! seems like quite the proposition! i'm down for whatever i can help to tackle in the development of the repository :)

**Comment by tedpatrick** [comment]
Contributed examples should be submitted as PRs to https://github.com/pyscript/pyscript-collective. We are focusing the examples within [PyScript](https://github.com/pyscript/pyscript) on feature use cases and testing.

Thanks for your patience, understanding, and contribution.

**Pull request closed without merging**